cmd/admin-heal.go

```diff
@@ -225,6 +304,12 @@ func mainAdminHeal(ctx *cli.Context) error {
 
 	// Heal format if bucket is not specified and quit immediately
 	if bucket == "" {
+		// If --fake was given, print all objects/uploads that
+		// need to be healed.
+		if isFake {
+			listAllToBeHealed(client, aliasedURL, isIncomplete)
```

Comment:
> @vadmeste while it would make if isFake { block shorter, this function doesn't fail (excl. the fatalIf), so it doesn't make sense to return error.

yeah, it is just little more readable to avoid calling fatalIf() inside listAllToBeHealed()